### PR TITLE
Fixed JSON export

### PR DESCRIPTION
**Pull request by GoComputing**
Added serialization to the classes 'ChatStore' and 'Messages' so that they can be JSON serialized.

### CODE REVIEW

**Comment by velecto1** [comment]
Hey, how can I try this, please?

I've just noticed this pull request after making the issue https://github.com/KnugiHK/Whatsapp-Chat-Exporter/issues/37.

I've already put your new lines into my repository and I've run `sudo pip install '.[android_backup]'` but I am still getting the same error message when running `wtsexporter` :(

**Comment by GoComputing** [comment]
Hello Tomáš,

When cloning my fork (where the updates are done,
https://github.com/GoComputing/Whatsapp-Chat-Exporter), change the the
'dev' branch. After that, execute the command (change the paths when needed)

wtsexporter -a -w ../Decrypted/wa.db -o HTML -j -d ../Decrypted/msgstore.db

Hope it helps.

Regards,
Carlos.

**Comment by velecto1** [comment]
Thank you, Carlos. Apparently, I was installing the original code GitHub repository as I left its URL i setup.py.

**Review by velecto1 — APPROVED** [review]
I've checked that the bugfix works; thank you, Carlos!

I don't really understand the purpose of commenting out `create_html(...)` on line 187 in `__main__.py`, but everything seems to work, including the creation of HTML files.

File: Whatsapp_Chat_Exporter/__main__.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ def main():
             messages(db, data)
             media(db, data, options.media)
             vcard(db, data)
-        create_html(data, options.output, options.template, options.embedded)
+        # create_html(data, options.output, options.template, options.embedded)
```
Comment (KnugiHK) [inline Whatsapp_Chat_Exporter/__main__.py:187]: Should this be uncommented

**Comment by KnugiHK** [comment]
> I've checked that the bugfix works; thank you, Carlos!
> 
> I don't really understand the purpose of commenting out `create_html(...)` on line 187 in `__main__.py`, but everything seems to work, including the creation of HTML files.

You mean even if create_html() is commented, the creation of HTML files still works?

**Comment by GoComputing** [comment]
I am so sorry, I shouldn't. I did when testing, and I forgot to remove it.
How can I make it correct? Should I create a new pull request?

**Review by KnugiHK — CHANGES_REQUESTED** [review]
Please uncomment line 187 in `__main__.py`

**Comment by KnugiHK** [comment]
> I am so sorry, I shouldn't. I did when testing, and I forgot to remove it. How can I make it correct? Should I create a new pull request?


I am not really sure, but I requested change and maybe you can uncomment it in your branch and maybe the new commit will be reflected in this pull?

**Comment by GoComputing** [comment]
> > I am so sorry, I shouldn't. I did when testing, and I forgot to remove it. How can I make it correct? Should I create a new pull request?
> 
> I am not really sure, but I requested change and maybe you can uncomment it in your branch and maybe the new commit will be reflected in this pull?

I uncommented the line and pushed. Now it is visible in the fix. Hope this is it.

**Review by KnugiHK — APPROVED** [review]
LGTM

**Comment by KnugiHK** [comment]
I also added a check before serialize the latest data model, so that the it is compatible with old model.

**Pull request merged**